### DISCUSSION TITLE
Need help with Quarto website

### DISCUSSION THREAD

**Discussion by mhollanders** [quarto-dev/quarto-cli#5520] [Q&A]
Hi all,

I'm a total Github noob and I'm pretty sure I messed something up with my website. I built a Quarto website and am hosting it via Github Pages but somewhere along the line I stopped being able to push changes to the github pages branch.

I don't even know how to ask the question so I'm asking someone to do it for me with financial compensation. If you're interested, please send me a DM (I don't even know if that's possible on Github).

Cheers,

Matt

**Comment by mcanouil** [comment]
Could you share your GitHub repository?
